configure.ac

```diff
@@ -4585,6 +4582,26 @@ AC_MSG_RESULT($HAVE_USABLE_WCHAR_T)
 # check for endianness
 AC_C_BIGENDIAN
 
+# Stuff for expat.
+case "$ac_cv_c_bigendian" in
+    yes)
+        BYTEORDER=4321
+        ;;
+    no)
+        BYTEORDER=1234
+        ;;
+    *)
+        AC_MSG_ERROR([unknown endianness $ac_cv_c_bigendian])
+        ;;
+esac
+AC_SUBST([BYTEORDER])
+
+AC_CHECK_FUNCS(memmove,
+  [DEFINE_HAVE_MEMMOVE='#define HAVE_MEMMOVE 1'],
+  [DEFINE_HAVE_MEMMOVE='#undef HAVE_MEMMOVE'],
+)
+AC_SUBST([DEFINE_HAVE_MEMMOVE])
```

Comment:
You may define AC_CONFIG_FILES([Modules/expat/expat_config.h]) here, to group things specific to Expat at the same place. It seems like AC_CONFIG_FILES() can be called multiple times.

Reply:
I prefer to keep all output files in one place